src/components/label-pill/LabelPill.scss

```diff
@@ -0,0 +1,54 @@
+@import '../../styles/variables';
+
+.bdl-LabelPill {
+    display: inline-block;
+    color: $bdl-gray;
+    background-color: $bdl-gray-10;
+    border: 1px solid;
+    border-color: $bdl-gray-20;
+    border-radius: 10px;
+
+    &.bdl-LabelPill--warning {
+        background-color: $bdl-yellorange-10;
+        border-color: $bdl-yellorange;
+    }
+
+    &.bdl-LabelPill--error {
+        background-color: $bdl-watermelon-red-10;
+        border-color: $bdl-watermelon-red;
+    }
+
+    &.bdl-LabelPill--success {
+        background-color: $bdl-green-light-10;
+        border-color: $bdl-green-light;
+    }
+
+    &.bdl-LabelPill--alert {
+        background-color: $bdl-watermelon-red-10;
+        border-color: $bdl-watermelon-red;
+    }
+
+    &.bdl-LabelPill--highlight {
+        background-color: $bdl-light-blue-10;
+        border-color: $bdl-light-blue;
+    }
+
+    &.bdl-LabelPill--info {
+        background-color: $bdl-light-blue-10;
+        border-color: $bdl-light-blue;
+    }
+}
+
+.bdl-LabelPill-content {
+    font-size: 10px;
+    line-height: 10px;
+    vertical-align: middle;
+
+    &.bdl-LabelPill-has-text {
+        padding: 3px 6px;
+    }
+
+    &.bdl-LabelPill-has-component {
+        padding: 3px 4px;
```

Comment:
I wonder if you could remove the icon/text special cases if you used flex to align the items instead of padding? ie. an icon would collapse to a circle and the content would be centered horizontally and vertically.